### ISSUE TITLE
Change background image issues

### ISSUE THREAD

**Issue by f3bandit** [Mte90/GBAATM-Rebirth#16]
I have tried changing the background, and selection bar images using just solid colored bitmaps. But all I get is a black screen at the code toggle page. do they need to be in a specific format, or bit depth?

**Comment by Mte90** [comment]
It is the right dimension?
About the bitmap I didn't investigated how the code works and what bit depth of image when I ported the code on this new UI.
I guess as the project was released around 2003 probably the most oldest bitmap format possible.

**Comment by f3bandit** [comment]
yeah 240x160 for rthe bg and 240x14 for the selector bar. just like it states on the buttons

**Comment by f3bandit** [comment]
I looked at the file in the source and it's 24bit, so no idea why it's not working?!?!

**Comment by Mte90** [comment]
The big issue of this project it is how understand how works.
I just ported the code to new UI and new C version but I have no idea how works internally...
Probably the bitmap is not converted rightly as there is a byte by byte manipulation.

**Comment by f3bandit** [comment]
Gotcha, looking forward to the next version, also cheat codes work flawlessly ty so much. it really is a great tool.
